src/opm/parser/eclipse/Parser/ErrorGuard.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+  Copyright 2019 Joakim Hove/datagr
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <iostream>
+#include <opm/parser/eclipse/Parser/ErrorGuard.hpp>
```

Comment:
These files are a general utility, and should therefore be located in opm/common/utility.

Reply:
Ehhh - that I do not agree; the current class/file is (at least yet) - very much geared toward `ParseContex` use and I think it should be located there.

Reply:
Ok.